### ISSUE TITLE
Publish admin tables migration?

### ISSUE THREAD

**Issue by isometriq** [z-song/laravel-admin#938]
Is it by design that the migrations are not published in the App migration folder ? At this moment, `php artisan admin:install` runs directly the migration in the package vendor folder, which is lean, yes.

However, I can't rollback, reset or refresh migrations because of that. `admin:install` does work, but whenever I need to refresh the database, I have to do inconvenient work (wipe run admin:install, migrate), otherwise it fails! My solution is to copy the migration file manually.

What do you think of copying migration file then `artisan migrate` instead of running it from package vendor folder during install ?

By the way, this package is amazing!

**Comment by isometriq** [comment]
While playing again, I've noticed that the admin user and menus are also generated (seed class) during installl.

So if you copy the migration file to the app's `database/migrations` folder.
It would make sense to copy also the seed class in the apps's `database/seeds` folder too.

Just like the `app/Admin` folder contents, the seeder can be customized.

If the app owner wants to use the admin seeder, probably always as a starting point, just need to use it in the main seeder `database/seeds/DatabaseSeeder.php`

What do you think?

**Comment by isometriq** [comment]
Ok, I've found a lean way so I don't need to copy the package seeder (although I may copy part of it to make my own menus later, or just make my own Menu model for a static menu)
```
<?php

use Illuminate\Database\Seeder;
use Encore\Admin\Auth\Database\AdminTablesSeeder;

class DatabaseSeeder extends Seeder
{
    /**
     * Run the database seeds.
     *
     * @return void
     */
    public function run()
    {
        // $this->call(UsersTableSeeder::class);
        $this->call(AdminTablesSeeder::class);
    }
}
```

**Comment by mr-cn** [comment]
This is really a problem

**Comment by z-song** [comment]
Fixed in version 1.5
